### PR TITLE
infra(node): update nvmrc

### PR DESCRIPTION
**Pull request by ST-DDT**
Update to [`iron`](https://nodejs.org/dist/latest-iron/) (Node 20).

Alternatively we could switch to [`hydrogen`](https://nodejs.org/dist/latest-hydrogen/) (Node 18).

(or should we remove that file?)

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-lts/gallium
+lts/hydrogen
```
Comment (ST-DDT) [inline .nvmrc]:
```suggestion
lts/iron
```

**Review by Shinigami92 — APPROVED** [review]

**Review by matthewmayer — APPROVED** [review]

**Review by xDivisionByZerox — APPROVED** [review]

**Comment by xDivisionByZerox** [comment]
Is this file still relevant? We introduced a dev containers environment in #1652, which allows developers to streamline the setup of the projects environment.

**Comment by ST-DDT** [comment]
> Is this file still relevant? We introduced a dev containers environment in #1652, which allows developers to streamline the setup of the projects environment.

Dev containers cause some issues for me when I try to commit due to win vs linux paths.
But are otherwise useful for me if I want to try a script on linux.
I think more people use nvm than devcontainers.

**Comment by ST-DDT** [comment]
Lets merge it for now and talk in the teammeeting which dev environment/tooling we want to support.

**Pull request merged**